protocol/app/ante/sigverify_test.go

```diff
@@ -25,160 +19,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestSetPubKey(t *testing.T) {
-	suite := testante.SetupTestSuite(t, true)
-	suite.TxBuilder = suite.ClientCtx.TxConfig.NewTxBuilder()
-
-	// keys and addresses
-	priv1, pub1, addr1 := testdata.KeyTestPubAddr()
-	priv2, pub2, addr2 := testdata.KeyTestPubAddr()
-	priv3, pub3, addr3 := testdata.KeyTestPubAddrSecp256R1(require.New(t))
-
-	addrs := []sdk.AccAddress{addr1, addr2, addr3}
-	pubs := []cryptotypes.PubKey{pub1, pub2, pub3}
-
-	msgs := make([]sdk.Msg, len(addrs))
-	// set accounts and create msg for each address
-	for i, addr := range addrs {
-		acc := suite.AccountKeeper.NewAccountWithAddress(suite.Ctx, addr)
-		require.NoError(t, acc.SetAccountNumber(uint64(i)))
-		suite.AccountKeeper.SetAccount(suite.Ctx, acc)
-		msgs[i] = testdata.NewTestMsg(addr)
-	}
-	require.NoError(t, suite.TxBuilder.SetMsgs(msgs...))
-	suite.TxBuilder.SetFeeAmount(testdata.NewTestFeeAmount())
-	suite.TxBuilder.SetGasLimit(testdata.NewTestGasLimit())
-
-	privs, accNums, accSeqs := []cryptotypes.PrivKey{priv1, priv2, priv3}, []uint64{0, 1, 2}, []uint64{0, 0, 0}
-	tx, err := suite.CreateTestTx(privs, accNums, accSeqs, suite.Ctx.ChainID())
-	require.NoError(t, err)
-
-	spkd := sdkante.NewSetPubKeyDecorator(suite.AccountKeeper)
-	antehandler := sdk.ChainAnteDecorators(spkd)
-
-	ctx, err := antehandler(suite.Ctx, tx, false)
-	require.NoError(t, err)
-
-	// Require that all accounts have pubkey set after Decorator runs
-	for i, addr := range addrs {
-		pk, err := suite.AccountKeeper.GetPubKey(ctx, addr)
-		require.NoError(t, err, "Error on retrieving pubkey from account")
-		require.True(t, pubs[i].Equals(pk),
-			"Wrong Pubkey retrieved from AccountKeeper, idx=%d\nexpected=%s\n     got=%s", i, pubs[i], pk)
-	}
-}
-
-func TestConsumeSignatureVerificationGas(t *testing.T) {
```

Comment:
Same question as above, why can we drop this test?

Reply:
Ditto as a above but in this case we don't have a fork of the `SigVerificationGasConsumer` ante handler.